docs/terminal/shell-integration.md

```diff
@@ -91,6 +91,35 @@ When using shell integration, it has a "quality" associated with it that declare
 
 To view the shell integration quality, hover the terminal tab. Optionally, select **Show Details** on the hover to view more detailed information.
 
+## IntelliSense (Preview)
+
+### Background
```

Comment:
We could remove this header

Reply:
☝️ Try to avoid empty headers - in this case, the background info can go directly under IntelliSense.